test/functional/apps/management/_scripted_fields.js

```diff
@@ -125,7 +122,7 @@ export default function ({ getService, getPageObjects }) {
           'painless',
           'number',
           null,
-          '1',
+          '100',
```

Comment:
I think this change was just to help test if the scripted field existing but was down below the view of the field list.  But it doesn't really matter if you change it back or not.

Reply:
Oh I remember rapping with you about that.  It was like "visibility" or something.
I'll leave it for now, but I'm glad we've this comment here, for posterity.